### ISSUE TITLE
Living Streets 2.0 Updates from Initial Staff Testing [LSRB]

### ISSUE THREAD

**Issue by KaroEngstrom** [cityofaustin/atd-data-tech#17993]
## Week of 6/24/2024 
- on Monday, we met with the LS Staff team who had been testing the 2.0 test environment (new applications, review, resubmittals -- previous records will be added later for further testing) 
- these are the notes from them for bug fixes and small updates
---
 
 ### Items to discuss in Sprint Review - no action for testing updates
 
![image](https://github.com/cityofaustin/atd-data-tech/assets/100615928/fc9c00fb-ff86-4b37-8622-fbd84d114148)

* Clarify the difference between the Sponsor and Ambassador at some point (and explain why/when a Sponsor is required) `Non-testing item; LS team can generate verbiage and indicate where they'd like it`
     * Solution options: Add this definition in a note somewhere in the Knack process. Maybe we add definitions on words via pop-ups when applicants hover over words

* On the “Activation Details” page, in Documents, add some of the information on what the activations will look like and what materials will be provided (signs, cones, etc) -`Non-testing item; LS team can generate verbiage and indicate where they'd like it`

**Comment by KaroEngstrom** [comment]
### EOI Submittal Form – Resident
➡️ **Next steps are confusing on this page:**
<img width="603" alt="unknown" src="https://github.com/cityofaustin/atd-data-tech/assets/100615928/1b0fd1db-1e28-41f5-8e7a-4ce1885ac78f">

* Solution options:
     * Make the above text ^^ more bold/colored (maybe in an orange box indicating “Applicant Instructions” like on the following page)
     * Better indicate that the button in “Details” can be clicked to move forward with the application

➡️**Clarify that Applicant will have to download and complete the Resident Approval form on the home page of the Applicant Portal** (I had to backtrack to figure that out)
     * Solution options:
           * Add to “Click to download the Resident Approval Form” on the Details page
           * Include instructions at the top of the page to download the form


➡️ **In the pop-up that appears when “Submit Resident Approval Form” appears, clarify what the  “Resident Approval Form Link” is**
     * Solution: ~~Either as a pop-up when applicants hover over words OR~~ under the words Resident Approval Form Link, include the text “In the case that an applicant collects signatures digitally, please link the PDF, Excel document, or Google Sheet where you collected signatures here”

* When the resident puts in time/date preferences on both HS and PS, make it match the recent change we made
     * Lowest priority - this change was made in PROD after TEST was created, and PROD reflects the correct date and validates for it

**Comment by KaroEngstrom** [comment]
### Review of EOI and transition into EOI Reviewed - Reviewer
* Change “Activation Number” to “Activation”
![unknown](https://github.com/cityofaustin/atd-data-tech/assets/100615928/3d4ff202-408a-4f4f-8e8f-12c20ae90efa)
* Can we reorder these buttons
![image](https://github.com/cityofaustin/atd-data-tech/assets/100615928/c66071b4-2c16-4462-a8fc-7c9b5eefd4e5)

**Comment by KaroEngstrom** [comment]
### Applicant recieves back EOI Reviewed - Resident
![unknown](https://github.com/cityofaustin/atd-data-tech/assets/100615928/1ee07860-5f1a-4da6-93e4-b82e4e0b5eab)
➡️ Lets not capitalize “details”
➡️ Change ‘live’ or ‘currently active’ version of this EOI/Application from GREEN to BLUE
➡️ Once the application is approved, then this button can change to GREEN

<img width="638" alt="unknown" src="https://github.com/cityofaustin/atd-data-tech/assets/100615928/64fb5d22-93b8-47aa-b28f-73e6ae4effe4">

➡️ Change “Activation #” to “Activation”
➡️ Move the “Submit” button to the end or bottom of the page

**Comment by KaroEngstrom** [comment]
### After Application has been approved or deferred – Resident
Resubmittal to be allowed dependent on the defined cohorts (Winter 2024 for current activations, Fall 2024 for the next cohort) << This needs to be configured - has not been decided yet
➡️  Add Cohort MC dropdown for now with Winter 2024 and Fall 2024 options for now

![Play Streets - 268  Mayor's Play Street](https://github.com/cityofaustin/atd-data-tech/assets/100615928/01685dd1-d4c4-427b-ac91-a315bffacc10)
➡️  Ensure the Resubmit button is available for applications which are approved as well as for those that are differed
➡️  When the person is resubmitting, under “Duration in Months”, we want to add text warning stating “The start date of the activation will depend on City schedule and budget constraints.”

**Comment by KaroEngstrom** [comment]
### Resubmittal Review – Reviewer
➡️  Ensure that the EAZ and applicant phone number (and all other relevant information) carries through on the reviewer side when applicant resubmits
- [ ] EAZ 
- [ ] Applicant Phone
 
![unknown](https://github.com/cityofaustin/atd-data-tech/assets/100615928/f3c2e9ab-058d-4273-83fb-442130065c4e)
![© To manage account connections for this applic](https://github.com/cityofaustin/atd-data-tech/assets/100615928/5fdbd76a-757d-46ff-90ce-09c3a361205e)

➡️   Ensure the ability to mark the applicant as an ambassador (and again the phone isn’t carrying through)
